### DISCUSSION TITLE
Request on deleting a deleted news report

### DISCUSSION THREAD

**Discussion by fitchch** [agorahub/blog0#14] [Q&A] [answered]
https://agora0.gitlab.io/pen/22q2/inmedia/2022/06/18/INMEDIA-%E5%AD%B8%E7%94%9F%E6%9C%83%E8%A7%A3%E6%95%A3%E5%BE%8C%E4%B8%8D%E7%8D%B2%E6%A0%A1%E6%96%B9%E7%BA%8C%E7%B4%84-%E4%B8%AD%E5%A4%A7%E7%94%9F%E6%8B%8D%E7%B4%80%E9%8C%84%E7%89%87%E9%81%93%E5%88%A5%E5%B1%B1%E5%9F%8E%E8%A7%92%E6%A8%82.html

The original page of this news has been deleted due to the interviewee's willingness. Please also delete this from your website to show respect.

**Comment by theagora** [comment]
@agorahub/pen

**Comment by hokoi** [comment]
香港噤聲效應日漸成型，應權衡新聞公開與潛在傷害擔憂。贊成協助撤銷公開訪問，儘管網絡蹤跡覆水難收。

**Comment by theagora** [comment, marked as answer]
### Follow-up Todos:
- [x] Step 1: Suggest remove as per the request  - 5 Jan, 2023
- [x] Step 2: Suggest remove as per #15 - 14 Feb, 2023
- [ ] Step 3: Collect ideas and opinions from community......
